deps/github.com/openshift/kubernetes/REBASE.openshift.md

```
@@ -404,10 +403,55 @@ them as the highest priority and release blockers for your team:
 
-## Updating with `git merge`
+## Updating with `redhat-chai-bot`
 
 *This is the preferred way to update to patch releases of kubernetes*
 
+[chai-bot](slack://app?team=T027F3GAJ&id=A0AJUKWDUR1&tab=messages) (ship-help-bot) is an internal tool has been given instructions to
+periodically check for upstream patch releases and complete the rebase autonomously. The steps taken are outlined below:
+
+
+1. Fetch upstream tags over the past 31 days:
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)